### ISSUE TITLE
Why do you annotate such traffic scene graph dataset? What's the application prospect?

### ISSUE THREAD

**Issue by gitzyong812** [Milomilo0320/A-Semi-automatic-Annotation-Software-for-Scene-Graph#5]
Hi, I have also done some work on scene graph. I am just wondering why you annotate such dataset.

**Comment by bibekyess** [comment]
Hi @zyong812. I guess annotating is needed to create a training dataset. Almost all of the existing scene-graph generation models need to be trained with images having ground truth relationship labels (with object labels and/or attribute labels too), so hand labeling the image manually takes a lot of time. In such case, this sort of annotation software comes handy. 
I hope it answers your doubt. :)
